mat-cypress/cypress/integration/measures/NewMeasureOptions.spec.js

```diff
@@ -0,0 +1,126 @@
+    cy.get(newMeasurePage.modelradioFHIR).click();
+    cy.get(newMeasurePage.generateCmsIdCheckboxLabel).should('contain','CMS ID');
+    cy.get(newMeasurePage.generateCmsIdCheckboxLabel).should('not.contain','eCQM ID');
+
+    cy.get(newMeasurePage.modelradioQDM).click();
+    cy.get(newMeasurePage.generateCmsIdCheckboxLabel).should('not.contain', 'CMS ID');
+    cy.get(newMeasurePage.generateCmsIdCheckboxLabel).should('contain', 'eCQM ID');
+  });
+
+  it('enables the Match CQL Library Name checkbox when Generate ID checkbox is checked', () => {
+    cy.get(newMeasurePage.generateCmsIdCheckbox).check();
+    cy.get(newMeasurePage.matchLibraryNameCheckbox).should('be.enabled');
+    cy.get(newMeasurePage.cqlLibraryName).should('be.enabled');
+
+    cy.get(newMeasurePage.generateCmsIdCheckbox).uncheck();
+    cy.get(newMeasurePage.matchLibraryNameCheckbox).should('be.disabled')
+    cy.get(newMeasurePage.cqlLibraryName).should('be.enabled');
+  });
+
+  it('disables and clears the CQL Library Name text box when the Match CQL Library Name Checkbox is checked', () => {
+    cy.get(newMeasurePage.generateCmsIdCheckbox).check();
+    cy.get(newMeasurePage.matchLibraryNameCheckbox).should('be.enabled');
+    cy.get(newMeasurePage.cqlLibraryName).should('be.enabled');
+
+    cy.get(newMeasurePage.matchLibraryNameCheckbox).check();
+    cy.get(newMeasurePage.cqlLibraryName).should('be.disabled');
+
+    cy.get(newMeasurePage.matchLibraryNameCheckbox).uncheck();
+    cy.get(newMeasurePage.cqlLibraryName).should('be.enabled');
+    cy.get(newMeasurePage.cqlLibraryName).type('ValidLibraryName');
+    cy.get(newMeasurePage.cqlLibraryName).blur();
+    cy.get(newMeasurePage.cqlLibraryName).should('have.value', 'ValidLibraryName');
+
+    cy.get(newMeasurePage.matchLibraryNameCheckbox).check();
+    cy.get(newMeasurePage.cqlLibraryName).should('be.disabled');
+    cy.get(newMeasurePage.cqlLibraryName).should('not.have.value');
+
+    cy.get(newMeasurePage.matchLibraryNameCheckbox).uncheck();
+    cy.get(newMeasurePage.cqlLibraryName).should('be.enabled');
+    cy.get(newMeasurePage.cqlLibraryName).should('not.have.value');
+  });
+
+  it('uses defaults on each new measure', () => {
+    cy.get(newMeasurePage.generateCmsIdCheckbox).should('be.enabled');
+    cy.get(newMeasurePage.matchLibraryNameCheckbox).should('be.disabled');
+    cy.get(newMeasurePage.cqlLibraryName).should('be.enabled');
+
+    cy.get(newMeasurePage.generateCmsIdCheckbox).check();
+    cy.get(newMeasurePage.matchLibraryNameCheckbox).check();
+
+    cy.get(newMeasurePage.cancelBtn).click();
+    cy.wait(1000);
+    cy.get(measureLibrary.newMeasureButton).click();
+    cy.wait(1000);
+
+    cy.get(newMeasurePage.generateCmsIdCheckbox).should('be.enabled');
+    cy.get(newMeasurePage.matchLibraryNameCheckbox).should('be.disabled');
+    cy.get(newMeasurePage.cqlLibraryName).should('be.enabled');
+  });
+});
+
+// describe('New Measure: Generate CMS/eCQM ID', () => {
```

Comment:
This was intentionally commented out. It needs to be moved to its own test suite since its testing beyond the New Measure UI.

Reply:
Removed since it wasn't going to be completed in this PR.